CMakeLists.txt

```diff
@@ -11,7 +18,7 @@ endif ()
 # for Windows only
 if (WIN32)
   set(CMAKE_CXX_FLAGS_RELEASE "-DNOMINMAX ${CMAKE_CXX_FLAGS_RELEASE} /MT")
-  set(CMAKE_CXX_FLAGS_DEBUG "-DNOMINMAX ${CMAKE_CXX_FLAGS_DEBUG} /MTd")
+  #set(CMAKE_CXX_FLAGS_DEBUG "-DNOMINMAX ${CMAKE_CXX_FLAGS_DEBUG} /MDd")
```

Comment:
When you comment it out, do the test build and run succesfully?

Reply:
Yes, test build ran successfully.

Reply:
Thanks